bench-plotter/src/bench_plotter/plotting/dashboard_processor.py

```
@@ -0,0 +1,1070 @@
+            if runs_data and not is_histogram_query(expr, panel_title, legend_format):
+                # Detect TPS-like panels and force overlay across modes
+                if is_tps_panel(panel_title, legend_format, expr):
+                    output_path = str(section_dir / f"{safe_panel_title}{filename_suffix}.png")
+                    create_tps_plot(
+                        runs_data, panel_title, legend_format, 
+                        section_dir, safe_panel_title, safe_legend, output_path, plot_title
+                    )
+                    continue
+                
+                if is_histogram_query(expr, panel_title, legend_format):
+                    print(f"Detected histogram query, processing as bar chart (overlaid modes)")
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)